gluster-exporter/main.go

```diff
@@ -15,11 +15,10 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-// ExporterVersion and GitSHA
+// exporterVersion, defaultGlusterd1Workdir, defaultGlusterd2Workdir and defaultConfFile
```

Comment:
Need to add some more information about these variable or we can remove this comment.

Reply:
fixed now